packet/flatcar-linux/kubernetes/calico/host-protection.yaml.tmpl

```diff
@@ -1,126 +1,109 @@
-# Customize inbound failsafe rules
-# https://docs.projectcalico.org/v3.5/getting-started/bare-metal/policy/failsafe
+# General
+# =======
+---
+# Disable default failsafe inbound rules
+apiVersion: crd.projectcalico.org/v1
+kind: FelixConfiguration
+metadata:
+  name: default
+spec:
+  failsafeInboundHostPorts: []
+
+# Host endpoints
+# ==============
+---
+${controller_host_endpoints}
+---
+${worker_host_endpoints}
+
+# Firewall policy
+# ===============
```

Comment:
Seems weird that this is part of the last yaml (because of separators), but no problem either... :)

Reply:
I don't understand. What is the change you are suggesting?

Reply:
I'd put the comment in the same yaml, i.e after the `---`. But this is a personal preference, please ignore it if you prefer :)

Reply:
But "Firewall policy" is a section title for multiple YAML documents...

Reply:
I agree with @johananl here, section titles like make sense IMO.

Reply:
Ohh, now I get it. Sorry, my bad! :)